extensions/amp-accordion/0.1/amp-accordion.js

```diff
@@ -171,6 +173,89 @@ class AmpAccordion extends AMP.BaseElement {
     this.currentState_[contentId] = !isSectionClosedAfterClick;
     this.setSessionState_();
   }
+
+  /**
+   * Handles accordion headers clicks to expand/collapse its content.
+   * @param {!Event} event Click event.
+   * @private
+   */
+  clickHandler_(event) {
+    // Need to support clicks on any children of the header.
+    let header;
+    for (let i = 0; i < this.headers_.length; i++) {
```

Comment:
if we install the handler on `header` is suggested above, `currentTarget` would always be the `header` element removing the need to loop. (except for up/down navigation support which I don't think we should do, see comment below)